src/components/debug_console/debug_console.tsx

```diff
@@ -40,14 +38,14 @@ export default class DebugConsole extends React.Component
         )
     }
 
-    resolve_render_view(): React.ReactNode {
+    resolve_render_view(typedef: Map<string, string>): React.ReactNode {
         if (this.state.show === false) return null;
-        const S = this.props.state as C0VM_RuntimeState;

```

Comment:
Also, I've figured out the problem with `C0VM_RuntimeState` in original code base.

Instead of casting `state` into `C0VM_RuntimeState` using `as`, we should use the abstract class `C0VM_RT` and only use the properties in that abstract class.